src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -792,6 +792,23 @@ function showCurrent(state)
         return $this->dumpHeader = preg_replace('/\s+/', ' ', $line).'</style>'.$this->dumpHeader;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function dumpString(Cursor $cursor, $str, $bin, $cut)
+    {
+        if ('' === $str && isset($cursor->attr['img-data'], $cursor->attr['content-type'])) {
+            $this->dumpKey($cursor);
+            // we can not do fancy CSS here since we are in a <pre>
+            $this->line .= "\n";
+            $this->line .= str_repeat(' ', $cursor->depth + \strlen('image: '));
+            $this->line .= sprintf('<img style="max-width:50em;max-height:50em;" src="data:%s;base64,%s" />', $cursor->attr['content-type'], base64_encode($cursor->attr['img-data']));
```

Comment:
the size of the image should be set using a class and the CSS above, rather than using inline styles. It makes things more flexible (tools using the VarDumper in an embedded way may customize the CSS) and more compatible with CSP (inline styles require a special whitelist, while the CSS could be shipped in a different way)

Reply:
Fixed, thanks